mod_reforged/hooks/config/strings.nut

```diff
@@ -1654,7 +1655,15 @@ foreach (vanillaDesc in vanillaDescriptions)
 				"The [Fatigue|Concept.Fatigue] Cost of weapon skills is reduced by " + ::MSU.Text.colorRed("25%") + "."
 			]
 		}]
- 	}),
+	}),
+	RF_SecondWind = ::UPD.getDescription({
+		Effects = [{
```

Comment:
We need Fluff for this perk. Ideas:
- `Just one more turn!` which is a reference to turn based games where you keep telling yourself that you will turn off the game after one more turn.
- `There\'s more where that came from."`
- `I\'m not done yet.`